Scripts/script-ExtractURL.yml

```diff
@@ -12,8 +17,17 @@ script: |-
   var filtered = ['http://schemas.microsoft.com/office/2004/12/omml', 'http://www.w3.org/TR/REC-html40'];
   var whitelist = getCSVListAsArray('Indicators Whitelist');
   while (found = reg.exec(text)) {
-      if (filtered.indexOf(found[0]) < 0 && whitelist.indexOf(found[0]) < 0) {
-          matches[found[0]] = true;
+      matches[found[0]] = true;
+      for (var i = 0 ; i < whitelist.length; i++) {
+          if (whitelist[i] === found[0]) {
+              delete matches[found[0]];
+              break;
+          }else{
+              if(whitelist[i].match("^~") && checkRegex(whitelist[i],found[0])){
```

Comment:
Changed except "startsWith()". Demisto doesn't recognise this command. I am getting this error: script failed to run: TypeError: 'startsWith' is not a function. So I left with the .match("^~") command.